src/main/java/uk/gov/cshr/domain/EmailUpdate.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.cshr.domain;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.time.Instant;
+import java.util.UUID;
+
+@Data
+@Entity
+public class EmailUpdate {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    private String code = UUID.randomUUID().toString();
```

Comment:
@m24murray random codes should always be url safe random strings, not guessible UUIDs.
```suggestion
    private String code = RandomStringUtils.random(40, true, true);
```